ftl/benchmark/args.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package defines the shared cli args for ftl binaries."""
+
+import argparse
+
+
+def base_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
```

Comment:
it might be worth using the same base parser as in common/args.py as most of the flags are the same.  Then you can make a sub parser here for just the benchmark related flags

Reply:
There are a few flags in common/args.py which aren't relevant to benchmarking, I feel like it might be more confusing to have extra flags that don't do anything?

Reply:
it's fine to keep as is.  The extra flags that don't do anything currently could be passed to the benchmark as well in the case that we at some point want to test them (as the benchmark is then just calling the builders with the flags passed).  This way if flags are added to the builders we only need to change them in once place.  Also if we wanted to test additional builder flags, they would already be passed to the benchmark.

Reply:
Ahh that makes sense, I'll make the change then. Thanks!